code/_globalvars/economy.dm

```diff
@@ -0,0 +1,16 @@
+GLOBAL_VAR(current_date_string)
+GLOBAL_DATUM_INIT(station_money_database, /datum/money_account_database/main_station, new())
+GLOBAL_DATUM_INIT(centcomm_money_database, /datum/money_account_database/central_command, new())
+
+GLOBAL_LIST_INIT(all_supply_groups, list(
+	SUPPLY_EMERGENCY,
+	SUPPLY_SECURITY,
+	SUPPLY_ENGINEER,
+	SUPPLY_MEDICAL,
+	SUPPLY_SCIENCE,
+	SUPPLY_ORGANIC,
+	SUPPLY_MATERIALS,
+	SUPPLY_MISC,
+	SUPPLY_VEND))
```

Comment:
If we have `SSeconomy` doesnt it make sense to throw these on there?

Reply:
yeah that would probably make more sense

Reply:
I will get to this soon